Tests/Acceptance/ClearExpiredTokensCommandTest.php

```diff
@@ -52,6 +56,27 @@ public function testClearExpiredAccessAndRefreshTokens(): void
         $output = $commandTester->getDisplay();
         $this->assertStringContainsString('Cleared 1 expired access token.', $output);
         $this->assertStringContainsString('Cleared 1 expired refresh token.', $output);
+        $this->assertStringContainsString('Cleared 1 expired auth code.', $output);
+
+        /** @var EntityManagerInterface $em */
+        $em = $this->client->getContainer()->get('doctrine.orm.entity_manager');
+        $em->clear();
+
+        $this->assertNull(
+            $this->client->getContainer()->get(AccessTokenManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_ACCESS_TOKEN_EXPIRED
+            )
+        );
+        $this->assertNull(
+            $this->client->getContainer()->get(RefreshTokenManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_REFRESH_TOKEN_EXPIRED
+            )
+        );
+        $this->assertNull(
+            $this->client->getContainer()->get(AuthorizationCodeManagerInterface::class)->find(
+                FixtureFactory::FIXTURE_AUTH_CODE_EXPIRED
+            )
+        );
```

Comment:
@X-Coder264 Are these assertions necessary since there are dedicated test for the managers:
- `DoctrineAccessTokenManagerTest`
- `DoctrineRefreshTokenManagerTest`
- `InMemoryAccessTokenManagerTest`
- `InMemoryRefreshTokenManagerTest`

I'd rather you add a `DoctrineAuthorizationCodeManagerTest` & `InMemoryAuthorizationCodeManagerTest` instead for consistency reasons.

Reply:
@HypeMC These assertions are needed, otherwise this test does not test anything beside the output which is useless and brings basically zero value IMO.

Reply:
I'll add the manager tests.

Reply:
@X-Coder264 Works for me.

Reply:
@HypeMC Tests added.